fw/http_parser.c

```diff
@@ -6916,6 +6928,13 @@ __h2_req_parse_authority(TfwHttpReq *req, unsigned char *data, size_t len,
 }
 STACK_FRAME_NON_STANDARD(__h2_req_parse_authority);
 
+void
+h2_set_hdr_accept(TfwHttpReq *req, const TfwCachedHeaderState *cstate)
+{
+	if (cstate->is_set && cstate->hdr_accept.text_html)
+		__set_bit(TFW_HTTP_B_ACCEPT_HTML, req->flags);
+}
```

Comment:
I think it makes sense to move both the new 2-line functions to `http_parser.h` to make then inlinable and avoid far calls

Reply:
I could do this, but that would require inclusion of some additional headers into http_parser.h as it is mostly uses forward declarations.